site/using.md

```diff
@@ -329,6 +329,13 @@ CONTROLLER                     STATUS   UPDATES
 default:deployment/helloworld  success
 ```
 
+# Releasing a locked controller
```

Comment:
It's not really the controller that's being released, but the image, so this should read "Releasing an image to a locked controller". I think it's also worth describing the circumstances in which you might want to do so, and pointing out that the controller remains locked afterward.